tests/functional/z_cluster/nodes/test_nodes_maintenance.py

```
@@ -610,3 +624,277 @@ def test_pdb_check_simultaneous_node_drains(
+                log.info(
+                    f"Finalizer: restarting "
+                    f"{len(self.stopped_node_objs)} stopped nodes"
+                )
+                try:
+                    self._nodes.start_nodes(self.stopped_node_objs)
+                except CommandFailed as e:
+                    log.error(f"Finalizer start_nodes failed: {e}")
+
+        request.addfinalizer(restart_stopped_nodes)
+
+    def _wait_for_stable_degraded(self, ct_pod, timeout=600, checks=3, interval=20):
```

Ruff (0.15.20)

[warning] Missing return type annotation for private function `_wait_for_stable_degraded`

Add return type annotation: `None`

(ANN202)